cmd/fortifyExecuteScan.go

```diff
@@ -294,12 +298,17 @@ func verifyFFProjectCompliance(config fortifyExecuteScanOptions, sys fortify.Sys
 
 	fortifyReportingData := prepareReportData(influx)
 	scanReport := fortify.CreateCustomReport(fortifyReportingData, issueGroups)
-	paths, err := fortify.WriteCustomReports(scanReport, influx.fortify_data.fields.projectName, influx.fortify_data.fields.projectVersion)
+	paths, err := fortify.WriteCustomReports(scanReport)
 	if err != nil {
 		return errors.Wrap(err, "failed to write custom reports"), reports
 	}
 	reports = append(reports, paths...)
 
+	err = fortify.UploadReportToGithub(scanReport, config.GithubToken, config.GithubAPIURL, config.Owner, config.Repository, config.Assignees)
```

Comment:
I guess we cannot assume that GitHub credentials are available for all projects ...
Might need to be checked here. Alternatively the error can be handled gracefully.

Generally, I guess we need a way to properly activate/de-activate the GitHub issue creation functionality.

Reply:
Shall we have an explicit switch or shall be base it on the availability of configuration parameters? What do you think? the majority of interactions as of now is for PR handling and therefore rather unrelated to creating issues in GH.

Reply:
just a thought: what about having a dedicated boolean parameter which gets fed by optimizedAndScheduled from cpe. Not 100% sure though if this is properly available.

Other than that I guess a dedicated boolean flag makes sense anyway. What do you think?